### ISSUE TITLE
callsigns are variable length

### ISSUE THREAD

**Issue by swharden** [swharden/FSKview#31]
consider logs like:

```
200712 1304  11 -21 -1.1  14.097076  OZ7IT JO65 37           0     1    0
200712 1312   2 -28 -1.5  14.097110  PA0ANH JO22 27         -1  1922    0
200712 1322   1 -28 -1.1  14.097110  <PA0ANH> JO22LH 27     -1  5495   -2
200712 1326   3 -27  0.6  14.097006  HB9FID JN47 37          0   306    0
200712 1326   8 -23 -1.0  14.097162  DG7RJ JN58 37           0   155    0
```

but the fixed-position calls expected here:
https://github.com/swharden/FSKview/blob/824d92e85640ecb7c816851bc7b264703a39c6d6/src/FSKview/WsprSpot.cs#L142

The solution is to capture this whole string, then split on the space character:
```
200712 1322   1 -28 -1.1  14.097110  <PA0ANH> JO22LH 27     -1  5495   -2
-------------------------------------^^^^^^^^^^^^^^^^^^^^^^--------------
```

**Comment by swharden** [comment]
Here's a full log file provided by Ko, NL9222
[ALL_WSPR (4).TXT](https://github.com/swharden/FSKview/files/4982801/ALL_WSPR.4.TXT)

**Comment by swharden** [comment]
resolved, will be in FSKview 1.0.8 uploaded in a few minutes...
